File: cpp/citrc/26l.cpp
#include<bits/stdc++.h>
using namespace std;
int n;
signed main(){
        cin>>n;
        if(n==0) return cout<<"Happy Birthday!\n",0;
        int t;
        while(n--){
                cin>>t;
                if(t%11==0) cout<<"How perfect this plate is!\n";
                else cout<<"Broken\n";
        }
}

File: cpp/citrc/26l.py
n=int(input())
if n==0:
    print("Happy Birthday!\n")
else:
    a=list(map(int,input().split()))
    for t in a:
        if t%11==0:
            print("How perfect this plate is!\n")
        else:
            print("Broken\n")

